loadProjectAndPopup.js

```diff
@@ -0,0 +1,329 @@
+const closeModal = () => {
+  document.querySelector('.modal-section-mbm').innerHTML = '';
+  document.getElementById('mb-modal-hide-show').style.display = 'none';
+};
+
+const projectArray = [
```

Comment:
- [ ] Please fix it so that your object contains appear when the project is launched. :wink:

Reply:
At my end everything is fine as shown in the screen shot above